### PR TITLE
DOC-442 | Authentication of notebooks

### PR DESCRIPTION
**Pull request by nerpaula**
The authentication of notebooks has changed. Changes reflected in the docs:
- updated screenshot
- adjusted step about accessing a notebook

### CODE REVIEW

File: arangograph/notebooks.md [hunk 1]
```diff
@@ -44,11 +44,16 @@ favorite GraphML libraries with GPUs.
...
-**Running**, the notebook's endpoint will be accessible.
-6. Click the **Open notebook** button to access your notebook. 
-7. To log in into your notebook, use the `root` password from your deployment.
+4. Enter a name and optionally a description for your new notebook. 
+5. Select a configuration model from the dropdown menu. Click **Save**.
+6. The notebook's phase is set to **Initializing**. Once the phase changes to
+**Running**, the notebook's endpoint is accessible.
+1. Click the **Open notebook** button to access your notebook. 
+2. To access your notebook, you need to be signed into ArangoGraph as a user with
+the `notebook.execute.execute` permission in your project. Organization
```
Comment (Robert-Stam) [inline arangograph/notebooks.md:53]: notebook.execute.execute --> notebook.notebook.execute

File: arangograph/notebooks.md [hunk 2]
```diff
@@ -44,11 +44,16 @@ favorite GraphML libraries with GPUs.
...
+owners have this permission enabled by default. The `notebook-executor` permission
```
Comment (Robert-Stam) [inline arangograph/notebooks.md:54]: IMO it should be: The `notebook-executor` permission --> The `notebook-executor` role

**Review by Robert-Stam — COMMENTED** [review]
see comments, further LGTM

**Comment by Simran-B** [comment]
Is there an API besides `internal-dashboard` that should be excluded from the permissions list?

Are the `dataloader-admin` and `mlservices-admin` roles something that should be excluded for now?

**Comment by Robert-Stam** [comment]
> > Are the `dataloader-admin` and `mlservices-admin` roles something that should be excluded for now?
> 
> AFAIK yes, these should be excluded for now. @Robert-Stam can you please confirm?

There are in BETA , so imo can be included

**Pull request merged**